dev-docs/source/MVPTutorial/PresenterExerciseSolution.rst

```diff
@@ -4,141 +4,126 @@
 Presenter Exercise Solution
 ===========================
 
-View
-####
+The View
+########
 
 .. code-block:: python
 
-    from qtpy import QtWidgets, QtCore, QtGui
+    from qtpy.QtCore import Checked, ItemIsEnabled, ItemIsUserCheckable, Unchecked
+    from qtpy.QtWidgets import QComboBox, QPushButton, QTableWidget, QTableWidgetItem, QVBoxLayout, QWidget
+    from typing import Union
 
+    TEXT_COLUMN = 0
+    WIDGET_COLUMN = 1
 
-    class View(QtWidgets.QWidget):
 
-        plotSignal = QtCore.Signal()
+    class View(QWidget):
```

Comment:
In the solution to this exercise, the `subscribe_presenter` function is missing on the view